integration-test/build.gradle.kts

```diff
@@ -283,7 +283,7 @@ tasks.test {
 
       // Gravitino CI Docker image
       environment("GRAVITINO_CI_HIVE_DOCKER_IMAGE", "datastrato/gravitino-ci-hive:0.1.7")
-      environment("GRAVITINO_CI_TRINO_DOCKER_IMAGE", "datastrato/gravitino-ci-trino:0.1.2")
+      environment("GRAVITINO_CI_TRINO_DOCKER_IMAGE", "datastrato/gravitino-ci-trino:0.1.3")
```

Comment:
Do you release a Docker image? If you release a Docker image, you should add the document in the https://github.com/datastrato/gravitino/blob/main/docs/docker-image-details.md

Reply:
Please do it. BTW, why do you need to upgrade trino ci version?